Meta/build-image-extlinux.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+SUDO="sudo"
+
+if [ "$(uname -s)" = "SerenityOS" ]; then
+    SUDO="pls"
+fi
```

Comment:
Maybe one for later, but can we `source` in common definitions instead of duplicating them across five different scripts?

Reply:
Same for `die` below.

Reply:
And, of course, `shellcheck` complains about me adding an include file with variable definitions, but it wasn't able to pick up that `SUDO` was never set in the earlier iteration of the script...